Modules/_datetimemodule.c

```diff
@@ -2792,9 +2905,11 @@ delta_total_seconds(PyObject *self, PyObject *Py_UNUSED(ignored))
     if (total_microseconds == NULL)
         return NULL;
 
-    datetime_state *st = get_datetime_state();
-    total_seconds = PyNumber_TrueDivide(total_microseconds, st->us_per_second);
+    datetime_state *st = GET_CURRENT_STATE(st);
 
+    total_seconds = PyNumber_TrueDivide(total_microseconds, CONST_US_PER_SECOND(st));
+
+    RELEASE_CURRENT_STATE(st);
```

Comment:
Here's an example of using the two macros related to module state lookup.

You may wonder why anything needs to be released.  It's because the lifetime of the module state is bound to the lifetime of the module.  Therefore, we avoid any possibility of a race on the lifetime of the module state by holding a local reference to the module as long as we need the state.  That is hidden by `GET_CURRENT_STATE()` and `RELEASE_CURRENT_STATE()`.